cmd/icingadb/main.go

```diff
@@ -74,9 +74,19 @@ func run() int {
 		}
 
 		logger.Info("Starting database schema auto import")
+
+		db, err := cmd.Database(logs.GetChildLogger("database"))
+		if err != nil {
+			logger.Fatalw("Can't create database connection pool from config", zap.Error(err))
+		}
+		db.SetMaxOpenConns(1)
```

Comment:
And, while on it, I doubt this limit is actually necessary.

Reply:
Necessary? No, I don't think so.

However, when this code is executed, the main database connection is also active with its 16 open connections - or whatever the user has configured. Thus, we would have 32 open connection now, which is twice the number of user-configured connections. In addition, importing the schema should be a linear procedure, actually being performed in one database transaction.